roles/init_dbserver/tasks/init_dbserver.yml

```diff
@@ -1,15 +1,33 @@
 ---
 
-# setting variables available for all nodes
-# any of the variables existing can be updated and configured
+- name: Set the os variable
+  set_fact:
+    os: "{{ ansible_distribution | replace('Red Hat Enterprise Linux', 'RHEL') }}{{ ansible_distribution_major_version }}"
```

Comment:
Do you think we should use ansible_facts[‘os_family’] == 'RedHat' and make changes accordingly?

Reply:
For Major version checks, we could use ansible_facts['distribution_major_version'] == "7"

Reply:
This code has just been moved away from the `main.yml` file. Maybe we should apply the changes you are suggesting in another PR, because this is not the subject of this PR.